website/content/api-docs/features/consistency.mdx

```diff
@@ -37,15 +37,101 @@ The three read modes are:
+  server agents other than the leader. Other requests will always be handled
+  by the leader no matter the number of Consul servers.
+  Only `stale` requests benefit from the enhanced read scalability provided by
+  [read replicas](/docs/enterprise/read-scale) and
+  [redundancy zones](/docs/enterprise/redundancy).
 
-Note that some endpoints support a `cached` parameter which has some of the same
-semantics as `stale` but different trade offs. This behavior is described in
-[agent caching feature documentation](/api/features/caching).
+
+### Intra-Datacenter Request Behavior
+
+The following diagrams show how the intra-datacenter request path varies
+per consistency mode and the relative trade-offs between level of consistency and performance.
+
+<Tabs>
+<Tab heading="Stale reads">
+
+[![stale consistency mode read request path](/img/consistency-modes/consistency-mode-stale.png)](/img/consistency-modes/consistency-mode-stale.png)
+
+</Tab>
+
+<Tab heading="Default reads">
+
+[![default consistency mode read request path](/img/consistency-modes/consistency-mode-default.png)](/img/consistency-modes/consistency-mode-default.png)
+
+</Tab>
+<Tab heading="Consistent reads, all writes">
+
+[![consistent consistency mode read, any write request path](/img/consistency-modes/consistency-mode-consistent.png)](/img/consistency-modes/consistency-mode-consistent.png)
+
+</Tab>
+</Tabs>
+
+### Cross-Datacenter Request Behavior
+
+When making a request across federated Consul datacenters, requests are forwarded from
+a local server to any remote server. Once in the remote datecenter, the request path
+is the same as a [local request with the same consistency mode](#intra-datacenter-request-behavior).
+
+[![consistency mode behavior across Consul datacenters](/img/consistency-modes/consistency-mode-cross-datacenter.png)](/img/consistency-modes/consistency-mode-cross-datacenter.png)
+
+## Usage
+
+### Consistency Mode Defaults and Overriding
+
+#### Consul DNS Queries
+
+When DNS queries are issued to [Consul's DNS interface](/docs/discovery/dns),
+Consul uses the `stale` consistency mode by default when interfacing with the
+underlying Consul service discovery HTTP APIs
+([catalog](/api-docs/catalog), [health](/api-docs/health), and [query](/api-docs/query)).
+To set an upper limit on how stale results are allowed to be,
+specify [`dns_config.max_stale`](/docs/agent/options#max_stale)
+in the configuration of the Consul agent serving DNS queries.
+
+If stronger consistency is required at the expense of increased latency,
+set [`dns_config.allow_stale`](/docs/agent/options#allow_stale) to `false`
+to make Consul use the `default` consistency mode when interfacing with the
+underlying Consul service discovery HTTP APIs.
+
+The consistency mode underlying Consul DNS queries cannot be overridden on a
+per-request basis.
+
+#### Consul HTTP API Queries
+
+All HTTP API read requests use the `default` consistency mode by default
```

Comment:
I should probably clarify that no write requests support defining a consistency mode, and that all writes behave using `consistent` consistency mode.